plugins/servicenow/plugin.spec.yaml

```diff
@@ -4,8 +4,8 @@ products: ["insightconnect"]
 name: servicenow
 title: ServiceNow
 description: ServiceNow is a tool for managing incidents and configuration management. Using the ServiceNow plugin for Rapid7 InsightConnect, users can manage all aspects of incidents including creation, search, updates, as well as monitor them for changes
-version: 7.4.1
-connection_version: 7
```

Comment:
Should this stay at 7?

Reply:
We have decided to move up the connection version after the last trigger change 🙂